[Bug][JAVA]: maven failed  to build libgar-jni.so

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

build the java sdk with maven got error:
```bash
[exec] /usr/lib/llvm-11/bin/clang++  -Dgar_jni_EXPORTS -I/home/john/GraphAr/java/src/main/native -I/home/j
ohn/GraphAr/java/src/test/native -isystem /home/john/GraphAr/java/target/native/graphar_ep-prefix/include -isystem /us
r/lib/jvm/java-8-openjdk-amd64/include -isystem /usr/lib/jvm/java-8-openjdk-amd64/include/linux  -flto -fforce-emit-vt
ables -O3 -g -std=c++17 -Wall -fPIC   -pthread -std=gnu++17 -o CMakeFiles/gar-jni.dir/target/generated-sources/annotat
ions/jni_com_alibaba_graphar_writers_builder_VerticesBuilder_cxx_0xd05edbb.cc.o -c /home/john/GraphAr/java/target/gene
rated-sources/annotations/jni_com_alibaba_graphar_writers_builder_VerticesBuilder_cxx_0xd05edbb.cc
[INFO]      [exec] make[2]: Leaving directory '/home/john/GraphAr/java/target/native'
[INFO]      [exec] make[1]: Leaving directory '/home/john/GraphAr/java/target/native'
[INFO]      [exec] make[2]: *** No rule to make target 'graphar_ep-prefix/lib/libgar.so', needed by 'libgar-jni.so'.  
Stop.
[ERROR] /home/john/GraphAr/java/build.xml:24: exec returned: 2
[ERROR] around Ant part ...<ant antfile="/home/john/GraphAr/java/build.xml">... @ 5:54 in /home/john/GraphAr/java/targ
et/antrun/build-main.xml
```


### Expected Behavior

build successful without error.

### Minimal Reproducible Example

maven clean test

### Environment

- Operating system: Ubuntu 20.04
- CMake version: 3.16.3
- GraphAr version: [commit](https://github.com/alibaba/GraphAr/tree/c3ced985b49a9c2be6aaff7acc287dc9a870cdec)


### Link to GraphAr Logs

_No response_

### Further Information

_No response_

--- a/java/CMakeLists.txt
+++ b/java/CMakeLists.txt
@@ -25,7 +25,7 @@ build_graphar_cpp()
 
 add_library(${LIBNAME} SHARED ${SOURCES})
 target_include_directories(${LIBNAME} SYSTEM BEFORE PRIVATE ${GAR_INCLUDE_DIR})
-target_link_libraries(${LIBNAME} ${CMAKE_JNI_LINKER_FLAGS} ${GAR_SHARED_LIB})
+target_link_libraries(${LIBNAME} ${CMAKE_JNI_LINKER_FLAGS} gar_shared)
 target_link_libraries(${LIBNAME} ${CMAKE_JNI_LINKER_FLAGS} Arrow::arrow_static)
 
 set_target_properties(${LIBNAME} PROPERTIES LINKER_LANGUAGE CXX)

